client/state/selectors/index.js

```diff
@@ -55,6 +55,7 @@ export getPostLikes from './get-post-likes';
 export getRawOffsets from './get-raw-offsets';
 export getReaderFeedsForQuery from './get-reader-feeds-for-query';
 export getReaderFollowedTags from './get-reader-followed-tags';
+export getReaderFollowForBlog from './get-reader-follow-for-blog';
```

Comment:
not true externals, no. rss feeds can have a blogID if they're connected to jetpack. those are the only ones that can have email subs.